kubernetes/model/object.c

```diff
@@ -4,28 +4,48 @@
 #include "object.h"
 
 object_t *object_create() {
-    object_t *object = malloc(sizeof(object_t));
+    object_t *object = calloc(1, sizeof(object_t));
 
     return object;
 }
 
 void object_free(object_t *object) {
+    if (!object) {
+        return ;
+    }
+
+    if (object->temporary) {
+        free(object->temporary);
+        object->temporary = NULL;
+    }
+
     free (object);
 }
 
 cJSON *object_convertToJSON(object_t *object) {
-    cJSON *item = cJSON_CreateObject();
+    if (!object) {
+        return NULL;
+    }
 
-    return item;
-fail:
-    cJSON_Delete(item);
-    return NULL;
+    if (!object->temporary) {
+        return cJSON_Parse("{}");
```

Comment:
Is this actually correct? Wouldn't `cJSON_Parse("null")` be more appropriate?

Reply:
"null" is more appropriate. I updated this.